app/src/main/java/org/wikipedia/readinglist/SortReadingListsDialog.kt

```diff
@@ -0,0 +1,91 @@
+package org.wikipedia.readinglist
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import org.wikipedia.R
+import org.wikipedia.activity.FragmentUtil.getCallback
+import org.wikipedia.databinding.DialogSortReadingListsBinding
+import org.wikipedia.databinding.ViewReadingListsSortOptionsItemBinding
+import org.wikipedia.page.ExtendedBottomSheetDialogFragment
+import java.util.*
+
+class SortReadingListsDialog : ExtendedBottomSheetDialogFragment() {
+    fun interface Callback {
+        fun onSortOptionClick(position: Int)
+    }
+
+    private var _binding: DialogSortReadingListsBinding? = null
+    private val binding get() = _binding!!
+    private var adapter: ReadingListSortAdapter? = null
+    private var sortOptions: List<String> = ArrayList()
```

Comment:
Can be `lateinit`, and no need to initialize here.

Reply:
Done.